config/environments/development.rb

```diff
@@ -31,6 +31,8 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.default_url_options = { :host => "localhost" }
```

Comment:
Is this in this PR for something in particular?

Reply:
It's required when I tried execute Rent Tests with User devise model.